modules/installation-aws-config-yaml.adoc

```diff
@@ -62,4 +66,6 @@ sshKey: ssh-ed25519 AAAA... <3>
 <2> If you do not provide these parameters and values, the installation program
 provides the default value.
 <3> You can optionally provide the `sshKey` value that you use to access the
-machines in your cluster.
\ No newline at end of file
+machines in your cluster.
+<4> To configure faster storage for etcd, especially for larger clusters, set the
+storage type as `io1` and set `iops size` to least `500`.
```

Comment:
iops and size are separate fields - iops=speed in I/O operations/second and size=how big disk is (Gb).   I suggest "To configure faster storage for etcd, especially for larger clusters, set the storage type as `io1` and set `iops` to 4000."